schema/deployment-config.json

```diff
@@ -176,9 +176,14 @@
         }
       }
     },
-    "TypedUuidForOmicronZoneKind": {
+    "OmicronZoneUuid": {
       "type": "string",
-      "format": "uuid"
+      "format": "uuid",
+      "x-rust-type": {
+        "crate": "omicron-uuid-kinds",
+        "path": "omicron_uuid_kinds::OmicronZoneUuid",
+        "version": "*"
+      }
```

Comment:
Worth noting that this change is identical on the wire, so there aren't any compatibility concerns.